slack_bolt/app/app.py

```diff
@@ -83,6 +84,8 @@ def __init__(
         name: Optional[str] = None,
         # Set True when you run this app on a FaaS platform
         process_before_response: bool = False,
+        # Set True if you want to handle an unhandled request as an exception
+        raise_error_for_unhandled_request: bool = False,
```

Comment:
The term `raise` is a bit Python-specific. If we introduce the same option to JavaScript and Java, using `throw` for them is a reasonable option.

Another option may be `unhandled_request_error_enabled`, which can be programming language agnostic.